client/app/telecom/telephony/alias/configuration/mode/easyHunting/telecom-telephony-alias-configuration-mode-easyHunting.html

```diff
@@ -47,222 +40,208 @@ <h4 data-translate="loading"></h4>
+                        <select id="toneOnClosing"
+                                name="toneOnClosing"
+                                class="form-control"
+                                data-ng-disabled="ModeEasyHuntingCtrl.isUpdating"
+                                data-ng-model="ModeEasyHuntingCtrl.optionsForm.toneOnClosing"
+                                data-ng-options="sound.soundId as sound.name for sound in ModeEasyHuntingCtrl.sounds"
+                                required>
+                            <option value="" data-translate="telephony_alias_configuration_mode_easyhunting_none"></option>
+                        </select>
+                        <button type="button"
+                                class="btn btn-default mt-2"
+                                data-ng-disabled="ModeEasyHuntingCtrl.managingSounds || ModeEasyHuntingCtrl.isUpdating"
+                                data-ng-click="ModeEasyHuntingCtrl.openManageSoundsHelper('toneOnClosing')">
+                            <span data-translate="telephony_alias_configuration_mode_file"></span>
+                        </button>
                     </div>
 
-                    <div class="clearfix mt-2">
-                        <div class="row">
-                            <button type="submit"
-                                    class="btn btn-primary float-right"
-                                    data-ng-disabled="ModeEasyHuntingCtrl.isUpdating"
-                                    data-ng-if="ModeEasyHuntingCtrl.hasChanges()">
-                                <span data-translate="telephony_alias_configuration_mode_easyhunting_change"></span>
-                                <spinner class="ml-1"
-                                         data-ng-if="ModeEasyHuntingCtrl.isUpdating"></spinner>
-                            </button>
-                            <button type="button"
-                                    class="btn btn-default float-right mr-1"
-                                    data-ng-disabled="ModeEasyHuntingCtrl.isUpdating"
-                                    data-ng-if="ModeEasyHuntingCtrl.hasChanges()"
-                                    data-ng-click="ModeEasyHuntingCtrl.undoChanges()">
-                                <span data-translate="cancel"></span>
-                            </button>
-                        </div>
+                    <div class="form-group">
+                        <button type="submit"
+                                class="btn btn-primary"
+                                data-ng-disabled="ModeEasyHuntingCtrl.isUpdating"
+                                data-ng-if="ModeEasyHuntingCtrl.hasChanges()">
+                            <span data-translate="telephony_alias_configuration_mode_easyhunting_change"></span>
+                            <bouncing-box-loader class="d-inline-block align-middle mr-2"
```

Comment:
Change order like `bouncing-box-loader` + `span` please. (Your `mr-2` will be ok then).